Fix Python netaddr tests so that they are always done in the same order and
therefore generate test results that we can successfully compare against a
master test results file.

This patch will be sent upstream.

--- netaddr-0.7.10/netaddr/tests/__init__.py.orig	2015-02-01 10:35:34.791261654 -0800
+++ netaddr-0.7.10/netaddr/tests/__init__.py	2015-02-01 10:44:21.299844898 -0800
@@ -38,7 +38,7 @@
         files = glob.glob(test_path)
         test_files.extend(files)
 
-    sys.stdout.write('testdir: %s\n' % '\n'.join(test_files))
+    sys.stdout.write('testdir: %s\n' % '\n'.join(sorted(test_files)))
 
     #   Add anything to the skiplist that we want to leave out.
     skiplist = []
